ajax/libs/winstrap/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "winstrap",
+  "filename": "css/winstrap.min.css",
+  "description": "Winstrap is the official Bootstrap theme for Microsoft design language.",
+  "version": "0.5.11",
+  "author": "Microsoft",
+  "license": "MIT",
```

Comment:
As @maruilian11 [said](https://github.com/cdnjs/cdnjs/pull/12028/files#r148291819), the license filed need to be modified to `licenses`, like 
```
  "licenses": [
    "MIT",
    "OFL-1.1"
  ]
```

Many thanks